traffic_ops/app/lib/API/Configs/ApacheTrafficServer.pm

```diff
@@ -1451,24 +1449,53 @@ sub logging_dot_config {
 			$text .= "	Format = '" . $format . " '\n";
 			$text .= "}\n";
 		}
+	}
+
+	for ( my $i = 0; $i < $max_log_objects; $i = $i + 1) {
+		my $log_filter_field = "LogFilter";
+		if ( $i > 0 ) {
+			$log_filter_field = $log_filter_field . "$i";
+		}
+		my $log_filter_name = $data->{$log_filter_field . ".Name"} || "";
+		if ( length($log_filter_name) > 0) {
+			my $filter = $data->{$log_filter_field . ".Filter"};
+			$filter =~ s/'/\\'/g;
+			$filter =~ s/\\/\\\\/g;
```

Comment:
This doesn't work. You escaped the quotes before the backslashes, so the backslashes added by the first step will be immediately escaped.

Reply:
Ugh -- good catch.  I'll take a look at the integration tests.